app/assets/v2/js/pages/onboard.js

```diff
@@ -11,6 +11,15 @@ $('.js-select2').each(function() {
   $(this).select2();
 });
 
+// removes tooltip
+$('#job select').each(function(evt) {
+  $('.select2-selection__rendered').removeAttr('title');
```

Comment:
This is not removing the titles I think because that element is dynamically created, maybe we can just disabled jquery-ui titles here.  
Changing to `$( ".select2-selection__rendered" ).tooltip( "destroy" );` will remove it